src/sql/workbench/contrib/resourceDeployment/browser/resourceDeployment.contribution.ts

```diff
@@ -19,13 +15,3 @@ CommandsRegistry.registerCommand({
 		return commandService.executeCommand('azdata.resource.deploy');
 	}
 });
-
-MenuRegistry.appendMenuItem(MenuId.ViewContainerTitle, {
```

Comment:
Is there a reason you have to remove this? I'd keep it there while adding other entry points. some users might find the feature missing.

Reply:
Good point. Users that are familiar with seeing deployment in the breadcrumb menu would expect to see it there. I'll bring it back.

Reply:
> Is there a reason you have to remove this? I'd keep it there while adding other entry points. some users might find the feature missing.

Do we have any data that indicates this is even being used from the breadcrumb?  It is not discoverable in this location.